김진/231211/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,26 @@
+function solution(lottos, win_nums) {
+  let zeroCount = lottos.filter((num) => num === 0).length; //알아볼 수 없는 번호 찾기
+  let sameCount = lottos.filter((num) => win_nums.includes(num)).length; //중복 요소 찾기
+  function resultFind(arr) {
+    switch (arr) {
+      case 6:
+        return 1;
+        break;
+      case 5:
+        return 2;
+        break;
+      case 4:
+        return 3;
+        break;
+      case 3:
+        return 4;
+        break;
+      case 2:
+        return 5;
+        break;
+      default:
+        return 6;
```

Comment:
```suggestion
      case 6:
        return 1;
      case 5:
        return 2;
      case 4:
        return 3;
      case 3:
        return 4;
      case 2:
        return 5;
      default:
        return 6;
```
저는 코딩 처음 시작할 때 filter나 map같은 메서드들을 잘 이해하지 못해서 활용을 못했는데 되게 잘활용해서 쓰시는 것 같아서 되게 좋네요.
근데 break는 빠져도 될 것 같아요.